src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx

```diff
@@ -51,7 +51,7 @@ function Service({
   const isEdit = !!identifier
 
   const updateChangeSource = useCallback(
-    (data: any, formik: FormikContext<MonitoredServiceForm>): void => {
+    (data: any, formik: FormikContextType<MonitoredServiceForm>): void => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
    (data: unknown, formik: FormikContextType<MonitoredServiceForm>): void => {
```